src/stdlib/File.ts

```diff
@@ -0,0 +1,87 @@
+import { Callable, ValueKind, BrsString, BrsBoolean } from "../brsTypes";
+import { Interpreter } from "../interpreter";
+import { Volume } from "memfs/lib/volume";
```

Comment:
Try to avoid importing things from other modules using their file path - it's entirely likely that they'd do a reorganization internally and not do a major version bump because the public interface has changed.  Consider following the patterns `memfs` recommends: https://github.com/streamich/memfs/blob/master/docs/reference.md

Reply:
The docs might not match the memfs library I ended up using.  The latest one had some missing dependencies.  

That makes sense, but what is the typescript type _getVolumeByPath_ would return?  Volume and vol are constructors, but not a type from what I can tell.  

```
// Volume is a constructor function for creating new volumes:
import {Volume} from 'memfs';

function getVolumeByPath(..): Volume {
}
```
This raises the following error:

> Return type of exported function has or is using private name 'Volume'.

I can simply remove the return by from _getVolumeByPath_, but I'm hoping we can keep that typing.

Reply:
You'd probably have to return `typeof vol` to get the compile-time type associated with that constructor.  It's a bit of a bummer, but I have a feeling `memfs`'s declarations use old-style constructor functions instead of classes.

Reply:
(updating here for posterity): @strattonbrazil and I talked out-of-band and agreed it's fine to reference by file path for now since there's a `yarn.lock` in place – especially when there's some discrepancies between the `memfs` docs and the implementation we're using.